code/modules/goonchat/browserOutput.dm

```diff
@@ -55,6 +57,18 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of ico
 	if(usr.client != owner)
 		return TRUE
 
+	//Spam check
+	if(world.time + (3 SECONDS) > last_check)
+		last_check = world.time
```

Comment:
Check is backwards? This will always be true.

Reply:
I have a tiny brain